src/plasoscaffolder/dal/sqlite_query_execution.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=no-member
+# pylint does not recognize connect and close as member
+"""Base for sql query validators"""
+import sqlite3
+
+from plasoscaffolder.dal import base_sql_query_execution
+
+
+class SQLQueryExecution(base_sql_query_execution.BaseSQLQueryExecution):
+  """Class representing the SQLite query validator
+  """
+
+  def __init__(self, database_path: str):
+    """Initializes the SQL Query Validator
+
+    Args:
+      database_path: the path to the sqlite database schema
+    """
+    self._connection = sqlite3.connect(database_path)
+    self._connection.isolation_level = None #no autocommit mode
+
+
+  def executeQuery(self, query: str) -> base_sql_query_execution.SQLQueryData:
+    """Executes the SQL query.
+
+    Args:
+      query (str): The SQL query to execute on the SQLite database.
+
+    Returns:
+      base_sql_query_execution.SQLQueryData: The data to the query"""
+    query_data = base_sql_query_execution.SQLQueryData()
```

Comment:
Design consideration:
* should a user be able to pass anything but a single SELECT query?